ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineExchangeTransitionConfiguration.java

```diff
@@ -70,8 +59,7 @@ public String getName() {
 
   @Override
   public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext) {
-    // update our QoS "last call time"
-    getQosTimer().resetTimer();
+    engineCallListener.executionEngineCalled();
```

Comment:
Minor point, we can DRY these up by adding this call into `ExecutionEngineJsonRpcMethod.response()` method rather than having it each engine api endpoint.

Reply:
The reason why I did not do that, was because of the unit tests. When I move it there, I am not sure if I can still test the correct behavior with the unit tests. Do you think it would still be possible?

Fabio had the same question:
https://github.com/hyperledger/besu/pull/4411#issuecomment-1251270287